src/amqp.c

```diff
@@ -73,11 +76,30 @@ const char * const QD_AMQP_COND_FRAME_SIZE_TOO_SMALL = "amqp:frame-size-too-smal
 const char * const QD_AMQP_PORT_STR = "5672";
 const char * const QD_AMQPS_PORT_STR = "5671";
 
+const char * const QD_AMQP_DFLT_PROTO = "tcp";
+
 int qd_port_int(const char* port_str) {
     if (!strcmp(port_str, QD_AMQP_PORT_STR)) return QD_AMQP_PORT_INT;
```

Comment:
I understand that this is not part of the pull request, but aren't these lines (with strcmp) redundant with the strtoul below?

Reply:
You're right. Maybe I should submit a new PR without the strcmp, as service name is now being resolved when strtoul returns 0. And also, using static values for amqp/amqps can also be a problem, in case someone changes the default values at /etc/services (no supposed to happen, but it is possible).